src/main/java/games/strategy/engine/data/GameMap.java

```diff
@@ -170,29 +154,6 @@ protected void addTerritory(final Territory t1) {
     m_territoryLookup.put(t1.getName(), t1);
   }
 
-  public void removeTerritory(final Territory t1) {
```

Comment:
Are you positive this isn't used anywhere by reflection or anything? I kind of wonder what this was written for.